### ISSUE TITLE
Create a Debian package for Montblanc

### ISSUE THREAD

**Issue by sjperkins** [ska-sa/montblanc#250]
It would be useful to have a debian [kernsuite](http://kernsuite.info/) package for montblanc.

In future, Montblanc will have a hard dependency on tensorflow 1.6 (and perhaps 1.7, as a release candidate has just come out). This in turn creates dependencies on

- CUDA 9.1
- cuDNN 7.0

CUDA 9.1 will be available in Ubuntu 18.04 bionic as a package. cuDNN 7.0 is a bit more tricky as I believe it is not redistributable (although debian packages are available [here](http://developer.download.nvidia.com/compute/machine-learning/repos/ubuntu1604/x86_64/).

Part of the reason that I'm creating this issue is so that we can keep a record of conversations regarding packaging. I think the cuDNN dependency has come up in previous discussions around packaging Montblanc and it would be good to remember these points.

**Comment by sjperkins** [comment]
The conda devs seem to have to work on somewhat similar problems:

https://github.com/conda-forge/staged-recipes/issues/654

https://github.com/conda-forge/conda-forge.github.io/issues/63

**Comment by gijzelaerr** [comment]
Yes, thanks for making this I priority. I would like to add that although it might not be possible at present time, it might be in the future. For that reason, I recommend keeping Montblanc compatible with Cuda 9.1 for now. But I know that also depends on what direction Tensorflow is going, which itself is a fast moving target.

**Comment by sjperkins** [comment]
One option might be to create a Montblanc package with a dependency on the CPU version of tensorflow, eliminating the CUDA and cuDNN dependencies.

**Comment by gijzelaerr** [comment]
would that still be useful?

**Comment by gijzelaerr** [comment]
related issue here https://github.com/kernsuite/packaging/issues/129
